### PR TITLE
Fix exports of babel-core

### PR DESCRIPTION
**Pull request by thejameskyle**

### CODE REVIEW

**Comment by loganfsmyth** [comment]
Needs more bytes, sorry. Can we add a linting rule for minimum file size?

**Comment by hzoo** [comment]
Needs some unit tests so we don't regress

**Comment by samlecuyer** [comment]
I hate to be _that_ guy, but `exports.guy` was an `Array` in the article.

**Comment by mpj** [comment]
Can you please extract this into a separate npm package? It seems like it could have general use.

**Comment by kentcdodds** [comment]
lol :joy:

**Comment by d3viant0ne** [comment]
I agree with @mpj - This should really be extracted into it a plugin for `Historical comedic value`

This may turn out to be the greatest medium post of all time :)
I wonder if the `Express/Loopback` group is going to have the same sense of humor  ...

**Comment by amasad** [comment]
Can you please use a gender neutral export name?

**Comment by raabbajam** [comment]
> Can you please use a gender neutral export name?

Umm, did you just assume it's gender? Lol, jk. `Guy` is his name, though. Guy Fieri.
Also, yes please fix it as array.
This one is better:
```
import {guy} from 'babel-core';
const [GuyFieri] = guy;
```

**Comment by amasad** [comment]
His name is offending my taste buds

**Comment by andreasklinger** [comment]
"Hate to be that guy"… but specs are missing

**Comment by stevemao** [comment]
Please remove that guy :(

**Comment by Daniel15** [comment]
This patch is insufficient. Here's the proper fix:

```
export const guy = [
'                                                                                                                                                                                                       
                                                                     `                  `                                                                                                               
                                                                         `              `   `                                                                                                           
                                                                    ``.```    . ``      `  `. `                                                                                                         
                                                                    ``````` ` ` `   ` . .  ``                                                                                                           
                                                                    ```. `.```.   ... ``..```` `                                                                                                        
                                                            `  `   `` .`.`` ..`.``....`.`.`....` .                                                                                                      
                                                            ` ..`  `. ..``.`....``....```.....``` `                                                                                                     
                                                      `    `.  ``.``.`.````.`.```...`.....``...````                                                                                                     
                                                          ` .  ...`,..`..```...``.````. `.```.`` .`                                                                                                     
                                                       .`  .````....`.````````````````````.````` ``                                                                                                     
                                                      . .` ``.`.`.``````````````````.``````````  `   `                                                                                                  
                                                     `.`` .```..`.````````````````````````````` `````                                                                                                   
                                                      `.` ...```.`````````````````````````````.``.`...                                                                                                  
                                                   ` ``..``````````````````````````````````````.`.``.`   `                                                                                              
                                                    ``````.`````````````````````````````````` ``..```  `.                                                                                               
                                                     ``..``````````````` .````````````````````.`..``` . `                                                                                               
                                               .`  `` .`.``.```.``````````.```````````````````.``...`.  `                                                                                               
                                                .``` .`.```````````````````````````````````````````.`. ``                                                                                               
                                               ` ```` ....````````````..````` ```````````````````````.`.  `                                                                                             
                                               ` ````..``````` ```````.`````` ````````` `````````..`..`.                                                                                                
                                              . ` `.  ``.```````````````````` ```.````` `````.```.``..`. .                                                                                              
                                               ` ``` `````` ``````````.``````````.````````.``.```.``..`.```                                                                                             
                                              `` ``.   ````````` ``````````````..`````````.```. `````.`..``                                                                                             
                                             `````.` ` ``````````.` ` ````````````````.` `.``.`````````. `.                                                                                             
                                             `   `.`  `````` ````.```````````````.`.```````````````````.``.`                                                                                            
                                           ```   `.````````` ``` ``````````````````````````````````````...``                                                                                            
                                            ```    `.`````` `````````.``````.````.``````````````````````.``.                                                                                            
                                            `.` `.`  ````````````.````````.`````````````````````` ````````.`                                                                                            
                                             ` ` ```   ``` ```  `.``````````.```.`..``..``.``````  `````.`..                                                                                            
                                              ``````. `````` `` ``````.``.``.`````..``````.````.`` ``````... `                                                                                          
                                              ```````..```````` ````.`.```````````..````````.```````````..``                                                                                            
                                              `.````` ` ````` `  `````.``.``````....``````.`.`..````````.```..                                                                                          
                                              `````.````  ``` ``  ````````.````.``..`.` ````.`.``.`````````.``                                                                                          
                                           ` `` `.`.`` `     ` ` ```.`````````````.``.``.`.`...``....``````.` `                                                                                         
                                             ``..```````` ` `. ``````...`````````````.`.`.`.....,,....````..`                                                                                           
                                            `.`,``````.```   `` ` .```..`````````.```````..,,...,,,,...```````                                                                                          
                                            `.`.`` ```````   ``````.`````````````````..`........,,,,...`````.````                                                                                       
                                             `.``.```````` ` ``.`..``.`....````````............,,,,,,..`````.                                                                                           
                                           ` ...`````````` `````.`.......`````````..`..`.......,,,,,,.`..`.`. `                                                                                         
                                            ``..`````````````.`..........`````````````........,.,,,,,........`                                                                                          
                                            ``````````````````.............````````````........,,,,,,.`.......                                                                                          
                                            .````. ``` `` `.....``.......`.````````````........,,,,,,,.....`.`                                                                                          
                                            `` `` ```` `````.............``````````````.......,,,,,,:,.......``                                                                                         
                                             ``.``  ``` ````................``````````.......,,,,,:,,,...`...`                                                                                          
                                           ` .`````  ``.````................`````````........,,.,,:,::,......`                                                                                          
                                           ``````````` `````.................```````.........,,,,,:,,:..`..,.`                                                                                          
                                            ```.``````` ```....................```..........,,,,,,,,::,.`.....`                                                                                         
                                            . `.``````..````.......,............``.........,,,,,,,,,,:,...,...                                                                                          
                                            `.  ``````.`.````.,.............................,,,,,,,:,:,....,..`                                                                                         
                                             `. `````..`.````.,...,........................,,,,,,,,,,::,...,..                                                                                          
                                            ``.``````....```..,,.,,,,......................,,,,,,,,,:::,,.,,.`                                                                                          
                                            `````````.``.```...,.,,,,,....................,,,,,,,,,,,::,,,.,,`                                                                                          
                                             `.``.`.``..``.....,.,..,,,....................,,,,,,,,,,:::,.,.,,                                                                                          
                                             .```.`..``.```.``.,.,,.,,,,,..................,,,,,,,,,,:::,,.,,`                                                                                          
                                            ` .```.........``..,,,,,,,.....................,,,,,,,,,,:::,,..,`                                                                                          
                                            `` `.````,`,`.```..,,,..,,,.......................,,,,,,,,:::,..,                                                                                           
                                             `````````.`..``...,,,.,,,........................,.,,,,,,:::;,.,.                                                                                          
                                             .`....````,..`.`.,,,,,,,,,........................,,,,,,::::',.,.                                                                                          
                                             ```.... ``.`..`.`,,,,,,,,.,.......................,,,,,,::::;:,,.                                                                                          
                                             ```.,..` ```.. `..,,,,,,,,.,......................,,,,,,,:::;;,:;.                                                                                         
                                             ``...,,,..``.`.`..,,,,,,,,.......................,,,,,,,:::::;,.;`                                                                                         
                                              ````,,.,,..`.`...,,,,,,,,..........`..........,,,,,,,,,,::::;,.:`                                                                                         
                                               `.``...`..```.`.,,,,,,,,....................,,,,,,,,,::::::;:,;.                                                                                         
                                               `.``.`.,``...`.,,,,,,,,,...................,,,,,,::;;;';;::::,;,                                                                                         
                                               ......`````...,:,,,,,,,,,..............,,,,,,:::'''''''''::::,:,                                                                                         
                                               `.....,.......,,,,,,,,,.,,......,,...,,,,,,::;;'++++++''';:::,,,                                                                                         
                                              `..````.......,:,,,,,,,,..,..,...,,,,,,,,,,,:;'++##++''';;;:::,,.                                                                                         
                                                ..``..`..,,:,,,,,,,,,,...,,,..,,:,,,,,,,,,:;'+++'';;;;::;::::.`                                                                                         
                                               `..,.`..,..,,:,,,,,,,,,,,.,,,::::::::,::,,:;'''';;;;;;;:::::::,`                                                                                         
                                               `.,,..,,,...:,,,,,,,,:,::::;;''';;::::,::::;;'''+####++;::::::,`                                                                                         
                                               ``.,:,,,.`..,,,,,,,,,::;;;''+++''';;::,,,,:;''++#@@@####;:::::,.`                                                                                        
                                                .,:;,,,,.:.:,,,,,,,:;''''++++++';;;::,,,:;'+'''@'@@#+++';::::,,                                                                                         
                                                ..';',,.:.,,,,,,,,:;;'''+''+';;;;;;;:,,,:;'+'###+';'';;'::,:::.                                                                                         
                                                 `:::;:,,...,,,,,,;;'''';;:;;;;;;;;;:,..,;+##';;:;;;::::::,,::`                                                                                         
                                                 ,::,,::....,.,,,,;;;;;;:::'#++;;;;;:...,;++';;:::::::;:,:,::;`                                                                                         
                                                 ,;;:,,.....,,.,,:;;;;:;;+@@@'+'';;':.`.,:''';:::::::;:,,,,::;                                                                                          
                                                 .;;':::....,,,,,:;;;;;+##@@#+###'';:.`..,;;;;;;;;;;;;:,,:,,:;                                                                                          
                                                 `;;+':,....,,..,:;;;;###@+#+'';;:;::.`..,,;:::;;;;;:.,,:,:::;`                                                                                         
                                                  ;;'';,,....,,,,,:;;+++';::;;::;;;,:....,,,::,,,:,,...,,::::;,                                                                                         
                                                  :;;'',..,...,,,,:;;;,:,,:,::;;;::,,......,:,,,.......,,,::,;:                                                                                         
                                                  :';;':....,,,,::::;:,,,,,,:;;;:::,.....,.,,,,........,,,::,;:                                                                                         
                                                  .;:;;:.,,,,,,,,:,.,,:,,:::;;::,:,,......,,,,,........,,,,:,;:                                                                                         
                                                   :;;;;,..,,,,,,...,.,:::;;;;:,,,,...``..,..,,........,,,,:,;:                                                                                         
                                                   :;;::,,,,,,,,,..,.,,::;;::,.,,,,...```..,,:,,........,,,,,:;                                                                                         
                                                   .:::;:.:,,,,,,,,.,,............,..`.``.,,:;:,,.......,,,,,:;                                                                                         
                                                    ,:::;:,:,,,,,,,.....``.`.........````..,;'';,......,,,,,,,:                                                                                         
                                                    :;,:;::,,:,,,,......````.`....,,,.```..,;;'',,,,,,.,,,,,,,'                                                                                         
                                                    ,:.:;:::,,,,,,.....````````...,:,,````.,;,,#':,,,,,,,,,,,,.,                                                                                        
                                                     ,,:;:;:::,,,,,....`````````.,:::,.```.,,.,:+;:,,,,,,,,,,,.,                                                                                        
                                                     ,:,,::::,:,,,.....````````.,:,,:,..``..,..,'';:,,,,:::,,:,`                                                                                        
                                                     ,,:,:::::,,,,......``````..::...,..``..,,,,'';;,,:,:::,::,`                                                                                        
                                                     ,.:,,::::,,,,.............,;:......```.,,,,''';:,::::::::,,                                                                                        
                                                      ,.,,::::,,,,............,:;:......``..,,,,''';::::::;::::'                                                                                        
                                                      ,..,::::,,,,,..........,:;::.........,,.:#''';;:::;;;;;;;`                                                                                        
                                                      ,.,,:::,,,,,..........,:;;::,,,,,...,,:;##';'+;:::;;;;;;:                                                                                         
                                                      `..,::::,,,,,,,.......::;;:::::::,,,,:;###':;+':::;;;;;;:                                                                                         
                                                       ..,::::,,,,,,,,,....,:;';::''';:::::;'##+;';'+;::;;;;;;:                                                                                         
                                                       :,,:::,,:,,,,,,,,...,:;;:::;'++';;;;;##+#''+++;;;;;;;;;,                                                                                         
                                                        ,,:::,,,:,,,,,,,,,,:;';;:;;+##++'''###++++'++';;;;;;;;.                                                                                         
                                                       `.:::::,:,:,::,,,,,::;';;;;;'+#+##++###++''++++;:;;;';;`                                                                                         
                                                       . `,:::::::::::::,,::;'';;;;'++++#+'+#+++':'+++;::;;;;;                                                                                          
                                                       ,  '::::::::::::::::;'''';;;''+++##'++'+';;++++;::;;;;'                                                                                          
                                                       :  +:::::::::;::::::;'+';;';'+++++++++''''#+'++;:;;;;;'                                                                                          
                                                       .  +:;::::::;:::;::;'+++''';'''+'++';;'';@#++++::;;';;:                                                                                          
                                                        :.+;;;:::::::;;:::;'+++''':;;;'';;;'';,;@#++++;:';;;'`                                                                                          
                                                        ,::;;;::;:::::;:::;++++''';;;;;;''';;::@##+'++;:;';;'                                                                                           
                                                         ':;;;;;;:;:;;;:::;+++#@;,,:,,,,.:.,::@@#+';'+:;;;;;'                                                                                           
                                                         #@';;;;:::::::;:::'+#+#@;,,.,.,`:,,'#@#+';;'+;;;;;',                                                                                           
                                                         +#:;;;;;;::::;::::;++'##@@#+#::.,'#####+';;'+'';;;'                                                                                            
                                                         ;@,;;;;;;;::;;:::::++';+#####++######++';;;;+''';;'                                                                                            
                                                         #@;';;;;;;::::::,,:;+;::'+##++++#+'#+'';;;:;++';;;,                                                                                            
                                                        `##@+;;;;;;::::;::,,;+::::;;'++''+++'';;;;;;;++'';;#@+                                                                                          
                                                        .;@#:';;';;:::::::,,;+;::,:::;;;'';;::;:;;:;;++';''@@@@@@@:`                                                                                    
                                                        ,+@@,';;;;;;::::;:,,;+;,,,,::,.,::,,:;;::::;;++';;'@@@@@@####@#;,;,`                                                                            
                                                        ,+#@+';;'';;;:::::::;+;:,::,::,,..,::;;::::;'++';;'@@@@@@@@@@#######@@@:`                                                                       
                                                        ,+###,';;'';;::::::,:+::,,::,::,,,,::;::::::;++';;'#@@@@@@##@##@@@@######@:`                                                                    
                                                        .+;##:';;'';;;::::::;+',,,:::,,,,,:::;,:::::'++';''@@@@@@@@###@#############@;  `                                                               
                                                        .'+#@#;;;;;;;;::::;::+':,,,,::,,,:,::,:,,:::'+#;;;'@@@@@@@@#####################@@                                                              
                                                        .':;@#,';;'';;:::;:::+'::,,,:::,,::,,...,,::'+#;;;'@@@@@@@@#######################@                                                             
                                                        `+@;+##'';;;;;:::::;;+'::.,,,,..,,,...,,,:::'+#';;'@#@@@@@#@#######################@                                                            
                                                        ;#@#'#@:';;;;;;;::::'+':,,,,,,....,....,,,::'+#';;'@#@@@@@#@########################+                                                           
                                                       ',+##+'@#:';;;;;:::::;++:,,,,,..,....,...,:,;'+++;;'@+#@@@@@@@########################'                                                          
                                                      `#;+#+#++#++;;;;;::::,;#'',...,...,,,..,`.,,:;++#+;;'@##@@@@###@########################'                                                         
                                                      ###:#'#+;@':+;;;;;::::;++':,.............,,,,;'###;;'@##@@@@@##@#########################'                                                        
                                                     #### #'##+@@',';;;;:::::#+;:,,.......`....:,.:;+##+';'@#@@@@@@###@#########################+                                                       
                                                    :####+:+@#+###+:;;;;;::,;+++;,,.,`,....`...,:,:'+#++;;+##+@@@@@##############################+                                                      
                                                    ###### ##'###@##;;;;;:::'##+':,,...`....,,.,:,;;###+';#:@##@@@@###############################+                                                     
                                                   +#######,#@###@@+#;;;;;:::+#+'::,..,,...,..,:::;'#+++';+:##+#@@@#################################                                                    
                                                  .########.#@###@@#'@;;;;;;;+##+;::,.,,.,..,,.,:;;+#+##+'+#+@'#@@@##################################                                                   
                                                  ########## @#@#@@##'@;;;;'++#++';;:,,,,,,,,,:,:'+##+#+''##,@;@@@@##################################.                                                  
                                                 ############,@###@@#++#;;;;'####''';:,,,,.,.,,:;'++###'';#@,#'@@@@@##################################                                                  
                                                .###########'.#####@##+##;'+######++';;,,.,,.,.;;'+#++''';#@;+@@@@@###################################.                                                 
                                                @##########';;`#@##@@##'@#'+#####+##+';;,,,,,,.;+++++'''';#@@;+##@@####################################;                                                
                                               ###########+;;;;;####@#''#@##########+#+':,.,:,,;++#+''''''+@@,#+#@@####################################@.                                               
                                              +##########+;;;;;;#@###@+''##############;;:,.::::+#''''''''+##:#;#@@####################################@#                                               
                                             :###########;;;;;;;;@###@@+'@@###+#+######'',,.:,;:++';'''';++@#+;:@@@@###################################@#,                                              
                                             ###########;;;;;;;;;;##@#@#'###@@#+++###+++'::,,,'';''''''''++#++.:#@@#######################################                                              
                                            +##########+;;;;;;;;;;:#@#@@++##@@@#@++####+';,,,,,;''''''''++'#;;@;@@########################################                                              
                                           `########+##;;;;;;;;;;;';###@#+##@@@@@;+';'+++;:,.,,;'''''''+++'##;#'@#@####################################@##                                              
                                          `###########+;;;;;;;;;;;;:'#@#@#+#@@@@@:#';;;;';:,,,:''''''''++''+#:####@####################################@##                                              
                                         .############;;;;;;;;;;;;;;,+#@###+@@@@@;#+';''';;;:,:''+''+'+++'++@@'###@#######################################,                                             
                                        `###########+';;;;;;;;;;;;;;;,'##@###@@@@#++'''''';,:,;'++++++++'''+#;;@##@@######################################@                                             
                                        ##########++#;;;;;;;;'';;;;;;','#@@###@@@@'#+''''';:::++++++++++'+'+;:,@##@########################################,                                            
                                       +############;;;;;;;;;''';;;;;;'.:#@###@@@@:#++''''';:+'####++++++'''#+.###@@######################################++                                            
                                       #########@###;;;;;;;;;;''';;;;;'',,#@###@@@:#++++++++##########+++'';+':#@#########################################+'.                                           
                                      '##+######@@@#;;;;;;;;;;;''';;;;;';,`#####@@'##+++++###########+++''';+:;@##@########################################''                                           
                                      ####+#######@#';;;;;;';;;;''';;;;;'':`##+#@@#+#################+++'';;#'########################################@#####'                                           
                                     :##############'';;;;;;;';;'''';;;;''';`:#+##@;#################++''';;+'###############################################:                                          
                                     ###########+###;'';;;';+;::,;''';;;;''''.`@##@,@##@@@@@@#########++';;;;,@######################################@#######:                                          
                                    #########+#+##++;'';;';;;''';;;''';;;;'''';`.@+.@@#@@@@@@@#######++''';;',###############################################+.                                         
                                   `#########++####;;;'+';;'';:;';'##+';;;'''''';`.,@@@@@@@@@@######+++'';;;+;################################################.                                         
                                   ###########+++#';;;;'';;;#':+,';+#'+';;;''''''';.`#@###@@@@######+++';;;;'#################################@###############'.                                        
                                  ;#############'';;;;;;;;;'+++;:+;,+#'';;;;''''''''+:`:+@##########++''';;;,@################################################+.                                        
                                  #############;;;;;;;;;;;;:'#+++;',,:#''';;'''''''''''+':,:+#@#####++'''';;.@###############################################'+'`                                       
                                 #############;;;;;;;;;;;;;;;+#;+++';;;++';;;''''''''''''++++;,;@##++''''';;:@################################################''`                                       
                                .##########+#;;;;;;;;;;;;;;;;;#;+'#+';'#'+';;;''''''''''''''++++'.'++''''';;'@################################################;'`,                                      
                                ###########++;;;;;;;;;;;;;';;;'+'++';,;'#''';;;'''''''''''''''+++++::'''';';#@##################################@####@########''`;:                                     
                               ;#########@##;;;;;;;;;;;;;;;';;'+#+###'::;'+';;;''''''''''''''''++'''+::'';;,@##################################@##############+''. `                                    
                               #####@@###+#;;;;;;;;;;;;;;+:+++''#####++::''+';;;''''''''''''''''''''''':;;;.###################################@@##############'''.                                     
                              '###@@######';;;;;;;;;;;;;;#+##+''######'+;'++'';;;;''''''''''''''''''''''':'.#@@################################@@##############'''+@.                                   
                              ###@@#######;;;;;;;;;;;;;;;;+##+''#######+':''+';;;;'''''''''''''''''''''''+:,##@################################@@##############+''+;,                                   
                             '#@@####@@@@@;;;;;;;;;;;;;;;+++''+++###+;;#++:;''';;;;;''''''''''''''''''''''+;#@@################################@@###############''';'                                   
                             #@########@@@#;;;;;;;;;;;;;;'''''#++++#++@;+++;:+';;;;;''''''''''''''''''''''+'@@#################################@@################'#+#`                                  
                            ,@@###########@+;;;;;;;;;;;;;;;;;++'+++;:#'#;'++;'+';;;;;;'''''''''''''''''''''##@#################################@@#################++#:                                  
                            #@#############+';;;;;;;;;;;;::::;+'+++;,#+#'++';''';;;;;;;''''''''''''''''''+;##@#################################@@##################'''                                  
                            @#############;;;';;;;;;;;;;;:::;+##'+';;+#@+@+#:;''';;;;''''''''';''''''''''':##@##################################@###################''                                  
                           +##############;;;;';;;;;;;;;;:::,'###'+;''++@#+##;''';;;;''''''''';;''''''''''.##@##################################@#################@#+'.                                 
                           @##############;;;;;';;;;;;;;;:,::;###++;':'+';++##;''';;;''''';;''';';''''''''`##@##################################@################@###';                                 
                          ;##############+;;;;;;;;;;;;;;;::,,;+####'',''+##+#';;'';;;;''''';''';;;''''''''`##@##################################@#############@##@####'                                 
                          @#############++;;;;;;'';;;;;;;:,,,:'##+#'+''''+#+#+;;;+';;;''''';;'';;;''''''''.##@@#################################@###############@#####+,                                
                          ###############';;;;;;;';;;;;;;:,:,,;++'###+:'''#;###;;'+;;;;;'''';;;;;;'''''''',#####################################@@##@##########@@######'                                
                         :##############+;;;;;;;;;';;;;;;:,,,::+#@####;'''';##+;;'+';;;'''''';;;;;'''''''''###@##################################@##@##########@########                                
                         @###############;;;;;;;;;;';;;;;;;::,,;+###'#+,'';'##+;;;'+;;;;;'''''';;;''''''';+###@##################################@##@#########@@########:                               
                         ################;;;;;;;;;;;;;;;;;#;:,,:+##+@##;'+:''#;';;''';;;''''''''''''''''',####@##################################@##@########@@##########                               
                        `###############+;;;;;;;;;;;';;;;;;;':,,;+##+;##';+'',#;;;;'+;;;;''''''''''''''''.####@##################################@##@########@###########                               
                        ,#############+#+;;;;;;;;;;;;';;;;'#:;,,:;##;####+'+''++:;;''';;;;;'''''''''''''' #######################################@##@#######@@###########,                              
                        ###############';;;;;;;;;;;;;;';;;;'#'';;#':'#;##+++'',+;;;;'+;;;;;''''''''''''''`#######################################@##@#######@############'                              
                        @##########@;;;;;;;;;;;;;;;;;;;';;;''+':::'+#:@###++;+;'';;;''';;';'''''''''''''':#####@#################################@#@#######@############+;                              
                        ##########+;;;;;;;;;;;;;;;;;;;;';;;';,:',;#+#++'###+:'';;';;###;;;;';''''''''''';'#######################################@@@####################+',                             
                       `########@'';;;;;;;;;;;;;;;;;;;;;';;:';;;:#+;';'@+@#++;+::;;;@##';;;;;''''''''''':######@#################################@#@######@##############'+                             
                       ;#######@;;;;;;;;;;;;;;;;;;;;;;;;;';;:'''''##'#+,#'';+:'+':;;+##+;;;;;'''''''';''`######@#################################@@@@####################'+                             
                       @#######;'';;;;;;;;;;';';;;;;;;;;;';;',;,+,;###;+;####+'+':;'+##+';;;''''''''''''`######@#################################@@@#####################';`                            
                       #######;;'';;;;;;;;;;;'''';;;;;;;;'';',,'':'+###;@:##;;++::;+###++;;;''''''''''''.#######@################################@@@######################;:'
]
```

**Comment by kaelig** [comment]
:shipit:

**Comment by jordanbuchman** [comment]
Porting to TempleOS may be sufficient, as it's text editor has embedded media support.

**Comment by Daniel15** [comment]
>  its text editor has embedded media support.

My computer appears to have a text editor with media support. It's called "Microsoft Word". It also lets you write your code in Comic Sans, which is really useful for professional enterprise-level SaaS codebases.

**Comment by amasad** [comment]
Comic sans is pretty SaaSsy

**Comment by valioDOTch** [comment]
This related issue gets way too few attention
https://github.com/defunctzombie/node-yummy/issues/7

**Comment by Daniel15** [comment]
Thanks for the fix.

:shipit:

**Comment by thejameskyle** [comment]
@hzoo @loganfsmyth can we get out an emergency bug fix? My team is blocked on this

**Comment by kittens** [comment]
I WANT TO GO TO FLAVOUR TOWN.

**Comment by kittens** [comment]
TAKE ME TO FLAVOUR TOWN. MERGED VIA 033681af8941d9678961f985c13e500c3c70f337.

File: packages/babel-core/src/api/node.js [hunk 1]
```diff
@@ -30,6 +30,195 @@ export function Plugin(alias) {
   throw new Error(`The (${alias}) Babel 5 plugin is being run with Babel 6.`);
 }
 
+// Sorry about all the MBs...
+/* eslint-disable max-len */
+export const guy = [
```
Comment (ev1stensberg) [inline packages/babel-core/src/api/node.js]: Hi, it's Babel Bulbasaur from SanFran Recruiting office. We saw this incredible Pull Request and want to offer you a full time job as a open source trainer. The job mainly consists of playing Pokemon and telling people that this practise is a good for having a strong codebase, please don't hesitate to contact us at: dontclosethisthread@sanfranrecruit.us

**Comment by AlastairTaft** [comment]
Wow, you actually merged it

**Comment by c9s** [comment]
...........................................................................................................

**Comment by darthrodolfo** [comment]
![] (http://gifrific.com/wp-content/uploads/2012/06/Boy-That-Escalated-Quickly-Anchorman.gif)

**Comment by OlegLustenko** [comment]
Well done :+1:

**Comment by ajwhite** [comment]
👎 you missed a dot, please review before submission

**Comment by dncrews** [comment]
ddd-ascii

**Comment by ultrafez** [comment]
Jokes are great (and I'm glad this was removed in #3646), but the fact that this was merged makes me seriously question the professionalism of this project's maintainers, considering how widely-used Babel is.

**Comment by jdan** [comment]
@ultrafez All jokes aside. I would give the maintainers a little credit here - they know what they're doing, and they care _deeply_ about their users. They're not going to cause anyone any harm for a joke. They just added a bit of flair to code that's never sent to the client, and I don't see anything particularly unprofessional about that.

**Comment by ajwhite** [comment]
It has been removed from the codebase, it was a joke that was not part of any version release - end users will not be affected.

**Comment by thejameskyle** [comment]
@ultrafez Thank you for your feedback, we take it very seriously and we'll continue to work to improve. Feel free to let us know if we're ever not meeting your standards in the future. It's our job to please you and if we ever fail to do that we want to hear about it.

Your humble servant,
James Kyle

**Pull request closed without merging**